pip/commands/list.py

```diff
@@ -91,6 +91,13 @@ def __init__(self, *args, **kw):
             help="Align package names and versions into vertical columns.",
         )
 
+        cmd_opts.add_option(
+            '--leaves',
+            action='store_true',
+            dest='leaves',
+            help="List packages that are not dependencies of installed packages.",
```

Comment:
OK, cool. BTW, "leaves" doesn't really feel like the right name for "packages that are not dependencies of installed packages". The term "leaves", to me, means "packages with no dependencies", which is what you implemented. So maybe as well as fixing the implementation, this option should have a more intuitive name?